src/telemetry.rs

```diff
@@ -216,7 +216,7 @@ impl Telemetry {
             setup()?;
 
             // Create a new agent from the InternalAgent struct
-            let agent = InternalAgent::new(TELEMETRY_IP, "443", &["telemetry.mithrilsecurity.io"]);
+            let agent = InternalAgent::new(TELEMETRY_IP, "443");
```

Comment:
Where do you use the telemetry URL?

Reply:
The URL is used in here :https://github.com/mithril-security/blindai/blob/telemetry/src/telemetry.rs#L163.